frontend/test/metabase/scenarios/admin/settings/localization.cy.spec.js

```diff
@@ -44,4 +40,55 @@ describe("scenarios > admin > permissions", () => {
     // that's why we have to assert on the x-axis, instead of a popover that shows on a dot hover
     cy.get(".axis.x").contains("April 25, 2016");
   });
+
+  it.skip("should display days on X-axis correctly when grouped by 'Day of the Week' (metabase#13604)", () => {
+    withSampleDataset(({ ORDERS }) => {
+      cy.request("POST", "/api/card", {
+        name: "13604",
+        dataset_query: {
+          database: 1,
+          query: {
+            "source-table": 2,
+            aggregation: [["count"]],
+            breakout: [
+              [
+                "datetime-field",
+                ["field-id", ORDERS.CREATED_AT],
+                "day-of-week",
+              ],
+            ],
+            filter: [
+              "between",
+              ["field-id", ORDERS.CREATED_AT],
+              "2020-03-02", // Monday
+              "2020-03-03", // Tuesday
+            ],
+          },
+          type: "query",
+        },
+        display: "bar",
+        visualization_settings: {
+          "graph.dimensions": ["CREATED_AT"],
+          "graph.metrics": ["count"],
+          "graph.x_axis.scale": "ordinal",
+        },
+      });
+    });
+
+    cy.visit("/collection/root");
+    cy.findByText("13604").click();
```

Comment:
I like this approach! It nicely avoids assuming a constant question id or pulling it out of the response body.